reduxApp2/dev/js/containers/fill-details.js

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {addUserDetails} from '../actions/index'
+
+
+class FillDetails extends Component {
+  constructor(props) {
+    super(props);
+    this.handleButton = this.handleButton.bind(this);
+  }
+
+
+  handleButton(event) {
+    var {myFirstName, myLastName, myEmail, myAddress, myNumber } = this.refs;
+    var userDetails = {firstName: myFirstName.value,
+                       lastName: myLastName.value,
+                       email: myEmail.value,
+                       address: myAddress.value,
+                       phoneNumber: myNumber.value}
+    this.props.addUserDetails(userDetails);
+  }
+
+
+  render() {
+    const {selectFilter} = this.props ;
```

Comment:
@VARTIKASINGH1234 semicolon should be just newar the 'this.props'. So it would be i.e 'this.props;'

Reply:
done